go.mod

```diff
@@ -63,6 +63,7 @@ require (
 	github.com/sirupsen/logrus v1.6.0 // indirect
 	go.uber.org/atomic v1.3.2 // indirect
 	go.uber.org/multierr v1.1.0 // indirect
+	golang.org/x/sync v0.1.0 // indirect
```

Comment:
This seems not correct. Why it's a indirect instead of a direct dependency?

Reply:
Because I added started using `errgroup` directly.